src/singleinstance.py

```diff
@@ -1,3 +1,4 @@
+#! /usr/bin/env python
```

Comment:
This is not a script. No shebang is needed here.

Reply:
Okay. Thanks